src/style.css

```diff
@@ -0,0 +1,121 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --bgcolor: #fff;
+  --border-color: #e1e7ed;
+  --text-color: #3d4e61;
+}
+
+.search__large {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+  background: var(--bgcolor);
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search__large:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search__large:focus-within {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid var(--border-color);
+  border-radius: 3px;
+}
+
+.search__large--image {
+  width: 19px;
+  height: 19px;
+  margin-left: 26px;
+}
+
+.search__large--text {
+  margin-left: 17px;
+  padding: 0;
+  font-family: Avenir, sans-serif;
+  line-height: 21px;
+  border: none;
+}
+
+.search__large--text::placeholder {
+  font-style: normal;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: normal;
+  color: var(--text-color);
+}
+
+.search__large--text:focus {
+  font-size: 16px;
+  font-weight: 800;
+  line-height: normal;
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+  outline: none;
+}
+
+.search__small {
+  display: flex;
```

Comment:
Follow the checklist, please . Fix it everywhere 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/99403198/476a2c3d-8175-46bc-91e0-697df059e2fa)